recipes-asteroid/asteroid-package-feed/asteroid-package-feed.bb

```diff
@@ -0,0 +1,30 @@
+SUMMARY = "Build community packages and create package index from the community packages and rootfs pacakges."
```

Comment:
I would reword that as "Create a package index from the packages included in rootfs and community packages"